stackdriver.go

```diff
@@ -104,6 +106,15 @@ type Config struct {
 	// will log additional information that is helpful when debugging errors.
 	// Optional. Defaults to false.
 	DebugLogs bool
+
+	//MonitoredResource identifies the machine/service/resource
```

Comment:
add a space between the `//` and the text to match formatting of the other fields.